src/TaskLineRenderer.ts

```diff
@@ -0,0 +1,228 @@
+import { Component, MarkdownRenderer } from 'obsidian';
+import type { Moment } from 'moment';
+import type { Task } from './Task';
+import * as taskModule from './Task';
```

Comment:
For my own education: What's the benefit of importing like this?
And is there a significance of `taskModule` begin with lower case?

Reply:
I find `import * as` useful for preventing namespace pollution. If I import individual classes from `Task` and use them straight out, e.g. `Status` or `Priority`, it can sometimes lose context to where it belongs, and `taskModule.Status` may be clearer.
I don't know what's the naming convention here, it seemed natural to me that an imported module is equivalent to an object so it begins with a lowercase (unlike a class for example), which is consistent with the common old Node.JS `require` clause.

Reply:
Oh wow, I spend a lot of time navigating to the location of a few functions with common names - seeing them called via the module/file name would really help comprehension.

I would go so far as to say that I would like to evolve the whole code base to that style. 

Related: I spend a lot of time futzing around adding or removing functions from import statements and that would go too.

I'm arbitrarily going to suggest that module names are capitalised like classes - as they feel more like classes and less like variables to me.